lldb/source/Core/AddressRange.cpp

```diff
@@ -203,3 +208,38 @@ void AddressRange::DumpDebug(Stream *s) const {
             static_cast<void *>(m_base_addr.GetSection().get()),
             m_base_addr.GetOffset(), GetByteSize());
 }
+
+bool AddressRange::GetDescription(Stream *s, Target *target) const {
+  const char *file_name = nullptr;
```

Comment:
Init this with "" so we don't need to set it. We need to do this because a valid target can be supplied, but it might not have the section loaded yet. But we can also have a AddressRange with no section, and below on line 221, we only set the filename to valid if there is a section, so this code can crash if we  have a target but no section as file_name will be NULL and crash in the `Printf(...)` call below.